scripts/jsign.mjs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {spawn} from 'child_process';
+import {resolve} from 'path';
+
+/**
+ * Get the full path of jsign.jar.
+ * @returns {string} The full path similar to <.../jsign.jar>.
+ */
+function getJsignJarPath() {
+  return resolve('third_party', 'jsign', 'jsign-4.0.jar');
```

Comment:
i'd personally include the dirname in this resolution so this script doesn't break if we move it, or call it from the wrong place - https://bobbyhadz.com/blog/javascript-dirname-is-not-defined-in-es-module-scope

Reply:
Good idea.